udfs/remote_udfs/nlp/README.md

```diff
@@ -0,0 +1,124 @@
+# Calling Google's Natural Language API Sentiment Analysis Example 
+
+Google's Natural Language API Sentiment Analysis inspects the given text and identifies the prevailing emotional opinion within the text, especially to determine a writer's attitude as positive, negative, or neutral.  
+
+To run this example, you will need the following APIs enabled:
+* [BigQuery API](https://cloud.google.com/bigquery/docs)
+* [Cloud Functions API](https://cloud.google.com/functions/docs)
+* [BigQuery Connections API](https://cloud.google.com/bigquery/docs/working-with-connections#enable_the_connection_service)
+* [Natural Language API](https://cloud.google.com/natural-language/docs/setup#api)
+
+## Running the example
+
+### Setting the environment variables 
+
+Replace the various environment variables below with your desired values.  
+
+**_NOTE:_** When setting the LOCATION, use a regional endpoint (for example, us-central1) for the purpose of this demo, the location will be used for both the Cloud Function and the BigQuery parameters.
+
+```
+PROJECT=your_project_id
+LOCATION=your_bigquery_dataset_location
```

Comment:
You're using this location as a region for deploying cloud function, but cloud functions are regional, and if someone selects US as location, then cloud function deployment will fail.

Either add a CLOUD_FUNCTION_REGION var or just deploy cloud function in us-central1